en/developer_guide/dashboard_widgets.rst

```diff
@@ -0,0 +1,66 @@
+Dashboard widgets
+=================
+
+The Back office dashboard is the place to show to the user information concerning him. It's like a
+blackoard on which are pinned post-it. Each post-it is a widget. Open Orchestra comes with four
+widgets:
+
+* My 10 latest published nodes
+* My 10 latest nodes
+* My 10 latest published contents
+* My 10 latest contents
+
+As a developper, you can customize your application dashboard by choosing widgets and/or create new
+ones. Here is how to do.
+
+Configuring the dashboard
+-------------------------
+To choose the widgets to display in the dashboard, add a key ``open_orchestra_backoffice.dashboard_widgets``
+to your app configuration. Then set the list of widgets you want to use. Here is an exemple in yaml
+format showing the 4 standard widgets:
+
+.. code-block:: yml
+
+    open_orchestra_backoffice:
+        dashboard_widgets:
+        - last_nodes
+        - draft_nodes
+        - last_contents
+        - draft_contents
+
+Creating a widget
+-----------------
+This operation is very simple. All you have to do is create a Backbone view and register it in
```

Comment:
may be a link to this documentation which explain how register a view
https://github.com/open-orchestra/open-orchestra-docs/blob/master/en/developer_guide/specific_backbone_view.rst